apps/condo/domains/meter/utils/taskSchema/AbstractMetersImporter.js

```diff
@@ -178,7 +178,14 @@ class AbstractMetersImporter {
                             const message = get(mutationError, ['message'])
                             const internalMessage = get(mutationError, ['extensions', 'message'])
                             const messageForUser = get(mutationError, ['extensions', 'messageForUser'])
-                            const rowErrors = [messageForUser || internalMessage || message]
+
+                            const originalError = get(mutationError, ['originalError', 'errors', 0, 'originalError', 'errors', 0])
+                            const originalMessage = get(originalError, ['message'])
+                            const originalInternalMessage = get(originalError, ['extensions', 'message'])
+                            const originalMessageForUser = get(originalError, ['extensions', 'messageForUser'])
+
+                            // We need to show as understandable error as possible
+                            const rowErrors = [originalMessageForUser || originalInternalMessage || originalMessage || messageForUser || internalMessage || message]
```

Comment:
Do we really need to pass internalMessage for user? I suppose it'll be quite unclear)

Reply:
Don't we have errors by default?

Reply:
Yes. Most errors happen when custom mutation try to create/update a model. For example, custom mutation trying to create a meter and the Meter model throws an error. In this case a have a understandable error from model wrapped by custom mutation error. So, I show the model's error if exists. I've added a test for such case.

Reply:
> Don't we have errors by default?

This errors lift up into the error file, so it's mandatory to has an error from model.